tests/v2/validation/provisioning/provisioning_node_driver_hybrid_test.go

```diff
@@ -99,26 +60,19 @@ func (r *RKE2NodeDriverProvisioningTestSuite) ProvisioningRKE2Cluster(provider P
 	}
 
 	tests := []struct {
-		name      string
-		nodeRoles []map[string]bool
-		client    *rancher.Client
+		name       string
+		nodeRoles  []map[string]bool
+		hasWindows bool
```

Comment:
So if hasWindows an argument here, why is there a need to have two setups for either hybrid or linux only?